infra/Freyfile.hcl

```diff
@@ -279,7 +279,7 @@ deploy {
     hosts = "uppy-server"
     name  = "Deploy uppy-server"
     roles {
-      role                          = "{{{init.paths.roles_dir}}}/deploy/1.4.0"
+      role                          = "{{{init.paths.roles_dir}}}/deploy/2.0.0"
```

Comment:
Although this is a major version update it should work without any modifications, see:

```diff
# diff -ruw roles/deploy/1.4.0 roles/deploy/2.0.0 | colordiff 
diff: roles/deploy/1.4.0/example/my-app/roles/local-ansistrano: recursive directory loop
diff -ruw roles/deploy/1.4.0/defaults/main.yml roles/deploy/2.0.0/defaults/main.yml
--- roles/deploy/1.4.0/defaults/main.yml	2017-01-12 11:23:57.090619919 +0100
+++ roles/deploy/2.0.0/defaults/main.yml	2017-01-12 16:19:38.000000000 +0100
@@ -12,15 +12,10 @@
 ansistrano_current_dir: "current"
 
 # Shared paths to symlink to release dir
-ansistrano_shared_paths: [ logs ]
+ansistrano_shared_paths: []
 
 # Number of releases to keep in your hosts, if 0, unlimited releases will be kept
-ansistrano_keep_releases: 10
-
-
-ansistrano_npm: no
-ansistrano_owner: "www-data"
-ansistrano_group: "www-data"
+ansistrano_keep_releases: 0
 
 # Deployment strategies variables
 
@@ -51,4 +46,4 @@
 
 ## Sends anonymous stats to the www.ansistrano.com servers
 ## You can disallow it by just setting this parameter to "no" in your playbook
-ansistrano_allow_anonymous_stats: no
+ansistrano_allow_anonymous_stats: yes
\ No newline at end of file
diff -ruw roles/deploy/1.4.0/tasks/symlink-shared.yml roles/deploy/2.0.0/tasks/symlink-shared.yml
--- roles/deploy/1.4.0/tasks/symlink-shared.yml	2017-01-12 11:23:57.090619919 +0100
+++ roles/deploy/2.0.0/tasks/symlink-shared.yml	2017-01-12 16:19:38.000000000 +0100
@@ -2,13 +2,13 @@
 # Ensure symlinks target paths is absent
 - name: ANSISTRANO | Ensure shared paths targets are absent
   file: state=absent path={{ ansistrano_release_path.stdout }}/{{ item }}
-  with_items: ansistrano_shared_paths
+  with_items: "{{ ansistrano_shared_paths }}"
 
 - name: ANSISTRANO | Ensure shared paths sources are present
   file: state=directory path={{ ansistrano_shared_path.stdout }}/{{ item }} mode="u=rwX,g=rwX,o=" recurse=yes owner={{ ansistrano_owner }} group={{ ansistrano_group }}
-  with_items: ansistrano_shared_paths
+  with_items: "{{ ansistrano_shared_paths }}"
 
 # Symlinks shared paths
 - name: ANSISTRANO | Create softlinks for shared paths
   file: state=link path={{ ansistrano_release_path.stdout }}/{{ item }} src={{ ansistrano_shared_path.stdout }}/{{ item }}
-  with_items: ansistrano_shared_paths
+  with_items: "{{ ansistrano_shared_paths }}"
```